en/docs/install-and-setup/setup/deployment-best-practices/basic-health-checks.md

```diff
@@ -30,6 +30,25 @@ curl -v http://localhost:9763/services/Version
 <ns:getVersionResponse xmlns:ns="http://version.services.core.carbon.wso2.org"><return>WSO2 API Manager-2.6.0</return></ns:getVersionResponse>
 ```
 
+When it comes to APIM Gateway, verifying the server status might not be sufficient. We need to verify that the APIs in the Gateway are ready to accept traffic. We can verify that the APIs have been deployed successfully using the Gateway health-check API. This will return 200 OK if **all the APIs** have been deployed successfully in the Gateway.
```

Comment:
When deploying the WSO2 API Manager Gateway, it's a best practice to ensure that all APIs are correctly deployed and ready to accept traffic before directing actual requests to the newly spawned Gateway. It's important to note that merely checking the server status might not be sufficient in this context.

To verify the successful deployment of APIs in the Gateway, we recommend using the Gateway health-check API. This dedicated API allows you to confirm the readiness of **all the APIs** in the Gateway. If everything is in order and all the APIs are deployed successfully, the health-check API will respond with a status code of 200 OK. This additional step helps guarantee that your API Gateway is fully prepared to handle incoming traffic.